File: src/Week241/T1.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var subsetXORSum = function(nums) {
    let n = nums.length;
    let ret = 0;
    let sub = [[]];
    for (let i = 0; i < n; i++) {
        let l = sub.length;
        for (let j = 0; j < l; j++) {
            sub.push([...sub[j], nums[i]]);
        }
    }

    for (const s of sub) {
        let t = 0;
        for (const i of s) {
            t ^= i;
        }
        ret += t;
    }
    return ret;
};

File: src/Week/Week236/T1.java
package Week236;

public class T1 {
    public int arraySign(int[] nums) {
        int ret = 1;
        for (int num : nums) {
            if (num < 0) {
                ret *= -1;
            }
            if (num == 0) {
                return 0;
            }
        }
        return ret;
    }
}
